pandas/core/groupby/groupby.py

```diff
@@ -1879,13 +1880,17 @@ def _agg_general(
         min_count: int = -1,
         *,
         alias: str,
-        npfunc: Callable,
+        npfunc: Callable | None = None,
+        post_process: Callable | None = None,
+        **kwargs,
     ):
         result = self._cython_agg_general(
```

Comment:
out of scope but it isnt obvious to me why we have both _agg_general and _cython_agg_general